docs/src/userguide/special_methods.rst

```diff
@@ -223,7 +223,7 @@ Depending on the application, one way or the other may be better:
 
   These constants can be cimported from the ``cpython.object`` module.
 
-* If you use the `functools.total_ordering<https://docs.python.org/3/library/functools.html#functools.total_ordering>`_
+* If you use the `functools.total_ordering <https://docs.python.org/3/library/functools.html#functools.total_ordering>`_
```

Comment:
This is an unrelated bit of invalid rst but I'm being lazy and fixing it as part of this PR.